typescript-loops-practice/filter-out-nulls.ts

```diff
@@ -0,0 +1,12 @@
+/* exported filterOutNulls */
+function filterOutNulls(values: any[]): any[] {
+  const noNull: any[] = [];
+  for (let i = 0; i < values.length; i++) {
+    if (values[i] === null) {
+      delete values[i];
```

Comment:
I don't think they wanted you to modify the original array, so the delete command here is not needed...